R/endpoints.R

```diff
@@ -14,6 +14,8 @@ api_build <- function(queue) {
             serializer = serializer_json_hintr())
   pr$handle("GET", "/meta/plotting/<country>", endpoint_plotting_metadata,
             serializer = serializer_json_hintr())
+  pr$handle("GET", "/hintr/version", endpoint_hintr_version)
+  pr$handle("GET", "/hintr/worker/status", endpoint_hintr_worker_status(queue))
```

Comment:
These endpoints are returning the response as an array i.e.
```
["{\"status\":\"success\",\"errors\":[],\"data\":{\"hintr\":\"0.0.7\",\"naomi\":\"0.0.7\",\"rrq\":\"0.2.0\"}}"]
```
Instead of
```
{\"status\":\"success\",\"errors\":[],\"data\":{\"hintr\":\"0.0.7\",\"naomi\":\"0.0.7\",\"rrq\":\"0.2.0\"}}
```

Should be fixed if you specify the serializer as `serializer_json_hintr()`.